source/grid/grid_tools_dof_handlers.cc

```diff
@@ -1384,9 +1377,6 @@ namespace GridTools
           }
       }
 
-    AssertThrow(best_cell.first.state() == IteratorState::valid,
-                ExcPointNotFound<spacedim>(p));
```

Comment:
In case we have never updated `best_cell`, does the resulting (invalid) iterator compare equal to `DoFHandler::end()`? In other words, I'm asking if the default constructed `DoFHandler::active_cell_iterator` is the same as `end()`. If not, we might need to initialize `best_cell.first` to `DoFHandler::end()`.